update.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "💾 Saving local changes if any."
+git stash
+
+echo "🪢 Retrieving the latest lively.next version."
+# Assumes that one runs this on `main` branch. Inside of lively, we ensure that the GUI trigger is not shown in other cases.
```

Comment:
Isnt it better to be safe than sorry? Why not check if we really are on main here.